src/classes/AnonymisePersonExtension.cls

```diff
@@ -78,6 +78,9 @@ public with sharing class AnonymisePersonExtension {
         }
```

Comment:
Methods should use the `lowerCamelCase` naming convention. I suggest to rename accordingly.